blog/test.py

```diff
@@ -103,6 +103,22 @@ def test_edit_article(self):
         self.assertEqual(article.content, '<p>another content</p>')
         self.assertEqual(result.status_code, 302)
 
+    '''
+    def test_edit_article_with_empty_data(self):
+        data = {
+            'email': 'emailtest@test.com',
+            'password': 'testtest',
+        }
+        self.client.post('/login', data=data, follow_redirects=True)
+        data = {'title': 'title', 'content': 'content'}
+        self.client.post('/article/write', data=data)
+        article = Article.objects.first()
+        data = {'title': '', 'content': ''}
+        result = self.client.post('/article/{}/edit'.format(str(article.id)),
+                                  data=data)
+        self.assertEqual(result.status_code, 400)
+    '''
```

Comment:
I have commented this tests in this WIP PR because I have an issue with the redirection in `blog/views`.